build-tools/script/diff2html.js

```diff
@@ -0,0 +1,61 @@
+const { exec } = require('child_process');
+const fs = require('fs');
+const inquirer = require('inquirer');
+
+const OUTPUT_DIR = 'dist/diff';
+const REPORT_NAME = 'templates.html';
+const EXT = '*.hbs';
+
+const questions = [
+  {
+    type: 'list',
+    name: 'type',
+    message:
+      'The script will generate diff reports for handlebars templates.\n What do you want to use as a diff to the current commit?',
+    choices: [
+      {
+        name: 'Commit hash',
+        value: 'hash',
+      },
+      {
+        name: 'Master branch',
+        value: 'master',
+      },
+    ],
+  },
+  {
+    type: 'input',
+    name: 'commitHash',
+    message: 'Please enter a commit hash',
+    when: answers => answers.type === 'hash',
+    validate: val => val.length > 0,
+  },
+];
+
+const createDir = () => {
+  if (!fs.existsSync(OUTPUT_DIR)) {
+    const path = OUTPUT_DIR.replace(/\/$/, '').split('/');
+
+    for (let i = 1; i <= path.length; i++) {
+      const segment = path.slice(0, i).join('/');
+
+      !fs.existsSync(segment) ? fs.mkdirSync(segment) : null;
+    }
+  }
+};
+
+inquirer.prompt(questions).then(answers => {
+  const diff = answers.type === 'hash' ? answers.commitHash : 'origin/master';
```

Comment:
Could we make `diff` a range (`from..to`), so we have more control over where to compare it to?

This also means we ask two questions (where the to defaults to `HEAD`).